src/demo-app/table/table-demo.html

```diff
@@ -167,6 +72,6 @@
 
   <md-paginator #paginator
                 [length]="_peopleDatabase.data.length"
```

Comment:
this would be wrong when filtering is applied, wouldn't it?

Reply:
Yes! Good catch, forgot to include some affordance for this. I imagine we could either directly change the pagination length in the data source since we have access to it, or offer some variable they could assign to the paginator's `[length]`. I think I'll go with the former and see how that works